proto/metapb.proto

```diff
@@ -52,6 +63,8 @@ message Store {
     int64 last_heartbeat = 11;
     // If the store is physically destroyed, which means it can never up again.
     bool physically_destroyed = 12;
+    // NodeState is used to replace StoreState which will be deprecated in the future.
```

Comment:
I think you can mark it as deprecated. See https://developers.google.com/protocol-buffers/docs/proto3#options.

Reply:
Yes, once the related feature is finished, I will mark the old state as deprecated.